From 30f5f3d2aa5ec1622eea4f16ad27f4fd7775d8bb Mon Sep 17 00:00:00 2001
From: Mike Snitzer <snitzer@redhat.com>
Date: Fri, 13 Jun 2014 14:35:46 -0400
Subject: [block] blk-mq: update a hotplug comment for grammar

Message-id: <1402670238-13916-59-git-send-email-snitzer@redhat.com>
Patchwork-id: 83814
O-Subject: [RHEL7.1 PATCH 058/150] blk-mq: update a hotplug comment for grammar
Bugzilla: 1105204
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

BZ: 1105204

Upstream commit 9fccfed8f0cad9b79575a87c45d6f5f6ee05bb66
Author: Jens Axboe <axboe@fb.com>
Date:   Thu May 8 14:50:19 2014 -0600

    blk-mq: update a hotplug comment for grammar

    Signed-off-by: Jens Axboe <axboe@fb.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/block/blk-mq.c b/block/blk-mq.c
index 2410e0c..3fdb097 100644
--- a/block/blk-mq.c
+++ b/block/blk-mq.c
@@ -1548,10 +1548,10 @@ static int blk_mq_queue_reinit_notify(struct notifier_block *nb,
  struct request_queue *q;
 
  /*
-  * Before new mapping is established, hotadded cpu might already start
-  * handling requests. This doesn't break anything as we map offline
-  * CPUs to first hardware queue. We will re-init queue below to get
-  * optimal settings.
+  * Before new mappings are established, hotadded cpu might already
+  * start handling requests. This doesn't break anything as we map
+  * offline CPUs to first hardware queue. We will re-init the queue
+  * below to get optimal settings.
   */
  if (action != CPU_DEAD && action != CPU_DEAD_FROZEN &&
      action != CPU_ONLINE && action != CPU_ONLINE_FROZEN)
-- 
1.7.1